tests/02_eradiate/02_system/test_mdistant_insitu.py

```diff
@@ -0,0 +1,120 @@
+import os
+
+import attrs
+import matplotlib.pyplot as plt
+import numpy as np
+import pytest
+import xarray as xr
+
+import eradiate
+from eradiate import unit_registry as ureg
+
+
+def compute(l="distant", sigma=1.0, rho=1.0, exp_cls="AtmosphereExperiment"):
+    result = []
+    spps = [4**i for i in range(0, 11)]
```

Comment:
Is there a special reason for using powers of four?

Reply:
I also made tries with the `ldsampler` to see if it would improve convergence (it doesn't) and that sampler requires a power-of-4 sample count.